### ISSUE TITLE
Images Broken

### ISSUE THREAD

**Issue by markwaters** [idno/known#1666]
## While trying to do this:
I moved my wifes known instance from olddomain.tld to newdomain.tld by importing the mysql database dump to the new server.
Also copied across the Uploads directory , checked the file and directory permissions (755).

## I encountered this error:
I could see that the images were still loading from the olddomain.tld url so I downloaded the images from the olddomain.tld site and re-uploaded the images to the newdomain.tld site.
The newly uploaded images are still loading from the old site , I checked by looking at the page details in Firefox's Page Info page and can see the olddomain.tld url.

## Some other notes:
Uploading new photos in new photo posts works perfectly.
Uploading previous photos in a new photo post works perfectly.

**Comment by mapkyca** [comment]
I think this is a migration issue, closing in favour of #480

**Comment by markwaters** [comment]
I have stumbled back into this issue again.
Today I setup a tor hidden service for my site.
I have copied the clearweb site.ini file for the hiddensite.ini , editing the KNOWN_DOMAIN and KNOWN_PORT to match the new address.
I am using the same database as the clearweb installation.
Everything works perfectly except the images which are being fetched from the clearweb site.
So it seems like the image URLs are written into the database rather than being set by the variable in the .ini file.
